lib/main.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:lavenir/models/user.dart';
-import 'package:lavenir/screens/shifts/add-availability.dart';
-import 'package:lavenir/screens/shifts/request-shift.dart';
-import 'package:lavenir/screens/shifts/view-shifts.dart';
+import 'package:lavenir/screens/shifts/availability.dart';
```

Comment:
Should we move the shifts folder stuff into the home folder? Or maybe just rename it?